R/esas-package.R

```diff
@@ -0,0 +1,7 @@
+#' @keywords internal
+"_PACKAGE"
+
+## usethis namespace: start
+#' @importFrom utils read.csv
```

Comment:
I assume you'll switch to `readr::read_csv()` eventually. If so, then this line is easy to forget (to remove). I would:

- List `utils` in dependencies
- Use `utils::read.csv()` wherever it is called now.